plugin/storage/es/options.go

```diff
@@ -75,6 +78,7 @@ func NewOptions(primaryNamespace string, otherNamespaces ...string) *Options {
 				BulkWorkers:       1,
 				BulkActions:       1000,
 				BulkFlushInterval: time.Millisecond * 200,
+				TagDeDotChar:      "@",
```

Comment:
What was the issue with using ':'?

From my ES past, I vaguely remember there being able to mark a field as not_analyzed so it was indexed but using the actual value without being processed. I see that this has been replaced by the concept of keyword in 5.x. Doesn't this avoid the issue of using '.' in the tag name?

Reply:
> What was the issue with using ':'?

It would have to be escaped. It's used in kibana query `tag.key:foo`. To avoid dots in field name we would have to set `enabled:false` on the field.